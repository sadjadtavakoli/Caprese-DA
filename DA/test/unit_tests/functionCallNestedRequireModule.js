let function_call = require('../inputs/simpleModule')

function_call.callee4()


