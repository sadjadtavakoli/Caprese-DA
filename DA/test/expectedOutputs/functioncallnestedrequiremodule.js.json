{"callee5-test/inputs/simplemodule.js-22-26":{"impacted":["t_1","t_7"]},"calleeWithArgs-test/inputs/simplemodule.js-29-31":{"impacted":["t_4"]},"calleeReturnValue-test/inputs/simplemodule.js-33-35":{"impacted":["t_4"]},"keyMap":{"t_1":"functioncallnestedrequiremodule.js-test/unit_tests/functioncallnestedrequiremodule.js-1-5","t_3":"simplemodule.js-test/inputs/simplemodule.js-1-39","t_4":"callee5-test/inputs/simplemodule.js-22-26","t_7":"calleeWithArgs-test/inputs/simplemodule.js-29-31","t_10":"calleeReturnValue-test/inputs/simplemodule.js-33-35"}}